src/Extensions/Abstractions/Activities/ActivityExtensions.cs

```diff
@@ -60,6 +60,24 @@ public static class ActivityExtensions
 		public static Activity SetResult(this Activity activity, TimedScopeResult result) =>
 			activity.SetTag(ActivityTagKeys.Result, ActivityResultStrings.ResultToString(result));
 
+		/// <summary>
+		/// MarkAsSuccess
```

Comment:
Please make more human readable, I suggest: Set activity result to Success. Similar for the other 2 methods.